en/docs/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/managing-apis-in-kubernetes.md

```diff
@@ -0,0 +1,168 @@
+                `--file` or `-f` : Location of the swagger file, zip file or the API project folder 
+
+        !!! example
+            ```go
+            apictl k8s update api -n PizzaShackAPI -f swagger.yaml
+            ```
+            ```go
+            apictl k8s update api -n PizzaShackAPI -f PizzaShackAPI_1.0.0.zip
+            ```
+            ```go
+            apictl k8s update api -n PizzaShackAPI -f PizzaShackAPI-1.0.0
+            ```
+
+    -   **Response**
+
+        ```go
+        creating configmap with swagger definition
+        configmap/pizzashackapi-swagger-1614042541 created
+        creating API definition
+        api.wso2.com/pizzashackapi configured
+        configmap "pizzashackapi-swagger" deleted
+        ```
+
+## Delete an API in Kubernetes cluster
+
+1.  Run the following command to delete an API in the Kubernetes cluster.
+
+    -   **Command** 
+
+        ```go
+        apictl k8s delete api -n <API Name>
+        ```
+
+        !!! info
+            **Flags:**
+
+            -   Required :  
+                `--name` or `-n` : Name of the API 
+    
+        !!! example
+            ```go
+            apictl k8s delete api -n PizzaShackAPI
+            ```
+
+    -   **Response**
+
+        ```go
+        api.wso2.com "pizzashackapi" deleted
+        ```
+
+## Generate Deployment Directory for Kubernetes
+
+1.  Run the following command to generate a sample directory with all the contents to use as the deployment directory when performing CI/CD pipeline tasks in Kubernetes.
+
+    -   **Command** 
+
+        ```go
+        apictl k8s gen deployment-dir -s <Path of Source directory> -d <Path of Destination directory>
+        ```
+
+        !!! info
+            **Flags:**
+
+            -   Required :  
+                `--source` or `-s` : Path of the source directory to be used when generating the directory. Source can be a API project folder or a zipped API project.
+            -   Optional :
+                `--destination` or `-d` : Path of the destination directory where the directory should be generated.       
+    
+        !!! example
+            ```go
+             apictl k8s gen deployment-dir -s PizzaShackAPI-1.0.0
+            ```
+            ```go
+             apictl k8s gen deployment-dir -s PizzaShackAPI-1.0.0 -d Deployment-Directory
+            ```
+
+    -   **Response**
+
+        ```go
+        The deployment directory for PizzaShackAPI-1.0.0 file is generated at Deployment-Directory
+        ```
+
+2.  You can add [kustomization.yaml]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/resources/kustomization.yaml), [kustomize-config.yaml]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/resources/kustomize-config.yaml) and project zip file.
+
+3.  Run the following command to generate the related config maps.
+
+    -   **Command** 
+
+        ```go
+        kubectl kustomize
+
+        ```
+
+
```

Comment:
Shall we remove the extra new line here?

Reply:
Fixed with 21f466ce5d99225a24a45351eae0997e87f8c2f5